fixtures/Bridge/Symfony/SymfonyApp/config/config_doctrine_phpcr.yml

```diff
@@ -18,6 +18,9 @@ doctrine:
                 password: ~
                 host: 127.0.0.1
                 port: 3307
+                charset: UTF8
+                options:
+                    1002: "SET NAMES 'UTF8' COLLATE 'utf8_unicode_ci'"
```

Comment:
required to workaround a bug related to the doctrine-phpcr integration (https://github.com/doctrine/DoctrinePHPCRBundle/issues/310) and keep tests green on lowest/highest supported versions

Reply:
Could you add that link to the command?

Reply:
Done

Reply:
Done